spring/boot2-actuator-autoconfigure/src/test/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfigurationInternalServiceTest.java

```diff
@@ -95,19 +96,123 @@ void exposeInternalServicesToManagementServerPort() throws Exception {
                       actuatorStatus = 404;
                       internalServiceStatus = 404;
                   }
+                  assertActuatorStatus(port, actuatorStatus);
+                  assertInternalServiceStatus(port, internalServiceStatus, settings, false);
+              });
+    }
+
+    @Nested
```

Comment:
Thanks for commenting. 
Since the test classes in this file have a different active profile and they are independent of each other, it would be better to just delete `@Nested`.